app/graphql/models/level.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Models
```

Comment:
This module should be named "Types"

Reply:
Here you got it right: 
https://github.com/Agilefreaks/FreaksCatalog-API/pull/7/files#diff-eb9a313cd462df6ec80d88d35d2eae1fddcd38136099d37d431738e500a350e0R4

All other Models:: should follow the same pattern. See how you are inheriting from `Types::BaseObject`